app/views/about/more.json.rabl

```diff
@@ -0,0 +1,10 @@
+node(:uri)                  { site_hostname }
+node(:title)                { Setting.site_title }
```

Comment:
`/app/views/api/v1/instances/show.rabl` uses Setting. for the 3 main bits of info too, which what I was referencing here - I thought that was the right way to do it.